addons/point_of_sale/models/pos_order.py

```diff
@@ -1105,6 +1108,7 @@ def _order_line_fields(self, line, session_id=None):
     refund_orderline_ids = fields.One2many('pos.order.line', 'refunded_orderline_id', 'Refund Order Lines', help='Orderlines in this field are the lines that refunded this orderline.')
     refunded_orderline_id = fields.Many2one('pos.order.line', 'Refunded Order Line', help='If this orderline is a refund, then the refunded orderline is specified in this field.')
     refunded_qty = fields.Float('Refunded Quantity', compute='_compute_refund_qty', help='Number of items refunded in this orderline.')
+    quantity_saved = fields.Integer(string="Quantity Saved", default=0)
```

Comment:
Remove this field. Use a copy in the front end rather than in the back end. This field will **always** be the exact same as the current quantity. So whenever you load the order, you create a copy  of the orderline quantity as the `savedQuantity` and you base whatever you do on that